client/app/scripts/superdesk-monitoring/monitoring.js

```diff
@@ -294,6 +294,13 @@
                 scope.cacheNextItems = [];
                 scope.cachePreviousItems = [];
                 scope.limited = (monitoring.singleGroup || scope.group.type === 'highlights') ? false : true;
+                scope.stageLookup = {};
+
+                if (scope.group.type === 'stage') {
```

Comment:
The ticket says Incoming Stage, but here it's working stage. Also, whats the strategy if user turns off the stage where the notification gets displayed?